From 592c957aafea1bd378ad93e1ee68286b5483407e Mon Sep 17 00:00:00 2001
From: monojp <martin@herndl.org>
Date: Fri, 6 Apr 2018 23:49:31 +0200
Subject: [PATCH 2/2] Exchange: correct the targeted SDK version to avoid
 permission fails otherwise not manually granted permissions lead to fc on
 account setup

Change-Id: I3fe87cc75d5152aa847b539cec7d5f60f7c5ed44
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index e5868f0a..f401d650 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -62,7 +62,7 @@
     <uses-permission
         android:name="com.android.email.permission.ACCESS_PROVIDER"/>
 
-    <uses-sdk android:targetSdkVersion="23" android:minSdkVersion="14" />
+    <uses-sdk android:targetSdkVersion="19" android:minSdkVersion="14" />
 
     <!-- additional uses -->
 
-- 
2.17.1

